__tests__/app.test.js

```diff
@@ -57,40 +57,78 @@ describe("/api/nonsense", () => {
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+          expect(response.body.article_id).toBe(1);
+          expect(response.body.author).toBe("butter_bridge");
+          expect(response.body.title).toBe(
+            "Living in the shadow of a great man"
+          );
+          expect(response.body.votes).toBe(100);
+          expect(typeof response.body.body).toBe("string");
+          expect(typeof response.body.topic).toBe("string");
+          expect(typeof response.body.created_at).toBe("string");
+          expect(typeof response.body.article_img_url).toBe("string");
+        });
+    });
 
-  test("GET 404: should return status code 404 and 'Id not found' when article_id is not found", () => {
-    return request(app)
-      .get("/api/articles/9999")
-      .expect(404)
-      .then((response) => {
-        expect(response.body.msg).toBe("Id not found");
-      });
+    test("GET 404: should return status code 404 and 'Article ID not found' when article_id is not found", () => {
+      return request(app)
+        .get("/api/articles/9999")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Article ID not found");
+        });
+    });
+
+    test("GET 400: should return status code 400, bad request for an invalid id ", () => {
+      return request(app)
+        .get("/api/articles/nonsense")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe(
+            "Bad request, invalid article_id/not a number"
+          );
+        });
+    });
   });
 
-  test("GET 400: should return status code 400, bad request for an invalid id ", () => {
-    return request(app)
-      .get("/api/articles/nonsense")
-      .expect(400)
-      .then((response) => {
-        expect(response.body.msg).toBe(
-          "Bad request, invalid article_id/not a number"
-        );
-      });
+  describe("PATCH", () => {
+    test("PATCH 200: should update the votes property of the article", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 10 })
+        .expect(200)
+        .then((response) => {
+          expect(response.body.article.votes).toBe(110);
+        });
+    });
+
+    test("PATCH 400: should return status code 400 for invalid inc_votes", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: "nonsense" })
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe(
+            "Bad request, inc_votes must be a number"
+          );
+        });
+    });
+
+    test("PATCH 404: should return status code 404 when article_id is not found", () => {
+      return request(app)
+        .patch("/api/articles/999")
+        .send({ inc_votes: 10 })
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Article ID not found");
+        });
+    });
   });
```

Comment:
Good to see you checked for a valid article id not found, how about an error test for an invalid is, such as `.patch('/api/articles/notAValidId');` ?